tests_integration/testing_utils/assertions/test_result_assertions.py

```diff
@@ -9,6 +9,10 @@ def assert_that_result_is_successful(result: CommandResult) -> None:
     assert result.exit_code == 0, result.output
 
 
+def assert_that_result_is_error(result: CommandResult) -> None:
+    assert result.exit_code == 1, result.output
```

Comment:
IMO we shouldn't hardcode exit code = 1.

We can have other error codes, for example `UsageError` class uses exit code = 2.

Reply:
👍 changed